src/D2L.CodeStyle.Analyzers/Immutability/ImmutableDefinitionChecker.cs

```diff
@@ -425,32 +431,47 @@ private enum AssignmentQueryKind {
 
 					return AssignmentQueryKind.Hopeless;
 
-				default:

```

Comment:
Moved this out of the `switch` so that any `InvocationExpression`s that are not in the list still get caught here.

Reply:
Huh? Just put that case above this?

Reply:
Oh breaks, I get it